app/gui2/src/components/GraphEditor/widgets/WidgetFunction.vue

```diff
@@ -306,9 +317,12 @@ export const widgetDefinition = defineWidget(WidgetInput.isFunctionCall, {
     // and to resolve the infix call as its own application.
     if (prevFunctionState?.callId === ast.id) return Score.Mismatch
```

Comment:
This check can now easily be combined with the `prefixCalls` one. Just make `callId` part of the `prefixCalls` set.

Reply:
Done